### DISCUSSION TITLE
Consistent On-Brand Artwork using Ideogram + OpenAI Assistants

### DISCUSSION THREAD

**Discussion by metaskills** [metaskills/experts#12] [Show and tell]
Demo repo here.

https://github.com/metaskills/unremarkable-ideogram-assistant

I blogged about this tool in more detail here.

https://www.unremarkable.ai/consistent-on-brand-artwork-using-ideogram-with-openai-assistants/

If I had a few days I would do a lot of things differently. Some raw ideas.

* Lean into multiple, concurrent tool calls. Change interfaces from arrays to objects.
* Force the open browser tool as a implementation of creative tool. Sometimes not called.
* Switch human interaction pattern. 1) Flipped Interaction Strategy

https://ai.gopubby.com/4-human-ai-interaction-patterns-for-experienced-chatgpt-users-9e49d4234013

**Comment by matheusmaiberg** [comment]
I would like to express my gratitude for the excellent work on the documentation for the Experts.js framework. As I am currently learning and understanding more about the framework, the information provided has been extremely helpful for my progress. The clarity and depth of the examples truly make a difference. Keep up the great work!
